ui/js/perf.js

```diff
@@ -5,15 +5,16 @@ var perf = angular.module("perf", ['ui.router', 'ui.bootstrap', 'treeherder']);
 treeherder.factory('PhSeries', ['$http', 'thServiceDomain', function($http, thServiceDomain) {
 
     var _getTestName = function(signatureProps, displayOptions) {
+        var suiteName = signatureProps.suite;
+        var testName = signatureProps.test;
+
         if (displayOptions && displayOptions.abbreviate) {
             // exclude "summary" for abbreviated output
-            if (signatureProps.test)
-                return signatureProps.suite + " " + signatureProps.test;
-            return signatureProps.suite;
+            return testName ? suiteName + " " + testName : suiteName;
```

Comment:
Could you make it so we apply the same logic if abbreviate is true? We use abbreviate in the alerts summary and want the same behaviour there.